### PR TITLE
update for new ci environment

### PR DESCRIPTION
**Pull request by jesterhazy**
*Description of changes:*

update for new ci environment

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-spark/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated the [changelog](https://github.com/aws/sagemaker-spark/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-spark/blob/master/README.md) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      - cd $CODEBUILD_SRC_DIR/sagemaker-spark-sdk
+      - sbt -Dsbt.log.noformat=true sbtVersion scalaVersion
+
+  pre_build:
+    commands:
+      # clone the integration test repo
+      - cd $CODEBUILD_SRC_DIR
+      - GITHUB_OAUTH_TOKEN=$(aws secretsmanager get-secret-value --secret-id /codebuild/github/oauth --query SecretString --output text)
+      - INTEGRATION_TEST_REPO_URI="https://${GITHUB_OAUTH_TOKEN}@github.com/jesterhazy/PRIVATE-sagemaker-spark-integ-tests.git"
+      - git clone --depth 1 --single-branch $INTEGRATION_TEST_REPO_URI integration-tests
```
Comment (yangaws) [inline buildspec.yml]: Why we cannot consume it as a codebuild source?

File: sagemaker-pyspark-sdk/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
 pyspark==2.3.2
 numpy
```
Comment (yangaws) [inline sagemaker-pyspark-sdk/requirements.txt]: Hope this works. Most cases when I see numpy it's bounded with versions.

Comment (jesterhazy) [reply sagemaker-pyspark-sdk/requirements.txt]: yes. this is pre-existing. i didn't want to make unnecessary extra changes as part of the ci change

File: buildspec.yml (hunk 1 above)
Comment (jesterhazy) [reply buildspec.yml]: we could, but then this project would not be able to share the same template or configuration as our other PR builds. this seemed better than setting this project up as a one-off or significant refactoring of the overall PR setup. 

plus, putting the integration tests into a separate project seems like a mistake that we should sort out in a future PR.

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://fasm6a6tm9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=cf6de2a4-4446-4c93-ada9-d27bad5b4212%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://fasm6a6tm9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=29a366e0-de8f-4c0f-b043-d35e8b2179b1%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://fasm6a6tm9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=2c704587-418e-4368-89dc-7b38a1e88b35%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**